modules/rosa-hcp-sts-creating-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,151 @@
+----
++
+.Example output
+[source,terminal]
+----
+ID                                USERNAME    STATUS
+2a7jli9n4phe6c02ul7ti91djtv2o51d  test-user   issued
+----
++
+[NOTE]
+====
+You can also view the credentials in a JSON output by adding the `-o json` argument to the command.
+====
+
+. To view the status of a break glass credential, run the following command, replacing <break_glass_credential_id> with the break glass credential ID:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c <cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+ID:                                    2a7jli9n4phe6c02ul7ti91djtv2o51d
+Username:                              test-user
+Expire at:                             Dec 28 2026 10:23:05 EDT
+Status:                                issued
+----
++
+The following is a list of possible `Status` field values:
++
+* `issued` The break glass credential has been issued and is ready to use.
+* `expired` The break glass credential has expired and can no longer be used.
+* `failed` The break glass credential has failed to create. In this case, you receive a service log detailing the failure. For more information about service logs, see _Accessing the service logs for Red Hat OpenShift Service on AWS clusters_. For steps to contact Red Hat Support for assistance, see _Getting support_.
+* `awaiting_revocation` The break glass credential is currently being revoked, meaning it cannot be used.
+* `revoked` The break glass credential has been revoked and can no longer be used.
++
+. To retrieve the `kubeconfig`, run the following commands:
+** Create a `kubeconfigs` directory:
++
+[source,terminal]
+----
+$ mkdir ~/kubeconfigs
+----
++
+** Export the newly generated `kubeconfig` file, replacing <cluster_name> with the name of your cluster:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME=<cluster_name> && export KUBECONFIG=~/kubeconfigs/break-glass-${CLUSTER_NAME}.kubeconfig
+----
++
+** View the `kubeconfig`:
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c mycluster --kubeconfig
+----
+.Example output
++
+[source,terminal]
+----
+apiVersion: v1
+clusters:
+- cluster:
+    server: <server_url>
+  name: cluster
+contexts:
+- context:
+    cluster: cluster
+    namespace: default
+    user: test-username
+  name: admin
+current-context: admin
+kind: Config
+preferences: {}
+users:
+- name: test-user
+  user:
+    client-certificate-data: <client-certificate-data> <1>
+    client-key-data: <client-key-data> <2>
+----
+<1> The client-certificate contains a certificate for the user signed by the Kubernetes certificate authorities (CA).
+<2> The client-key contains the key that signed the client certificate.
++
+. Optional: To save the `kubeconfig`, run the following command :
++
+[source,terminal]
+----
+$ rosa describe break-glass-credential <break_glass_credential_id> -c mycluster --kubeconfig > KUBECONFIG
```

Comment:
Sorry but last change from my side, actually before it was giving error when used $, but now, it can be used, otherwise new KUBECONFIG file will be created. Thanks
```suggestion
$ rosa describe break-glass-credential <break_glass_credential_id> -c mycluster --kubeconfig >$ KUBECONFIG
```